lib/urllib_kerberos/__init__.py

```diff
@@ -34,24 +34,21 @@
     import kerberos as k
 except ImportError:
     if sys.platform == 'win32':
-        import kerberos_sspi as k
+        import winkerberos as k
     else:
         raise SystemExit("Could not import kerberos library. Please ensure "
                          "it is installed")
 
-
 log = logging.getLogger("http_kerberos_auth_handler")
 
 
 class AbstractKerberosAuthHandler:
     """auth handler for urllib2 that does Kerberos
        HTTP Negotiate Authentication"""
 
-    def __init__(self, password_mgr=None):
+    def __init__(self, mech_oid, password_mgr):
```

Comment:
By inserting mech_oid in before password_mgr, it could break current application which use AbstractKerberosAuthHandlers directly -- though I don't know if that something to be concerned about.